### PR TITLE
Add about_PowerShell_NuGet_Packages

### PR DESCRIPTION
**Pull request by rjmholt**
# PR Summary
Adds an about topic describing the various NuGet packages available for PowerShell.

We should evaluate whether this document or mutations of it should go in older PowerShell version directories.

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles (where are these?)

## PR Checklist

- [x] I have read the [contributors guide](https://docs.microsoft.com/powershell/scripting/community/contributing/overview) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

**Comment by sdwheeler** [comment]
Should this be an about topic? Is this really useful viewed in the console from the command line?

I suggest this should be in the SDK docs. Or find a better place in the TOC.

**Comment by rjmholt** [comment]
Are the SDK docs currently those under https://github.com/MicrosoftDocs/PowerShell-Docs/tree/staging/reference/docs-conceptual/developer?

That documentation tree seems to be very Windows PowerShell focused. Is there any documentation under there presently that refers to PS 6+? What's the best way for us to introduce PS 6+ information under that documentation segment?

**Comment by sdwheeler** [comment]
@rjmholt I agree, the developer content is very outdated. It needs to be rewritten. 

@joeyaiello (and others) We should have a meeting to discuss the best place to publish this. I am open to ideas.

**Comment by daxian-dbw** [comment]
@sdwheeler @joeyaiello Can we publish this documentation to somewhere similar to https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7? This write-up is extremely useful to developers of PowerShell and should be made visible as soon as possible.

File: reference/7.1/Microsoft.PowerShell.Core/About/about_PowerShell_NuGet_Packages.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+keywords: powershell,cmdlet,nuget
+ms.date: 09/27/2019
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_pipelines?view=powershell-7.x&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Pipelines
```
Comment (sdwheeler) [inline reference/7.1/Microsoft.PowerShell.Core/About/about_PowerShell_NuGet_Packages.md:6]: The URL and the title refer to about_Pipelines.

Comment (rjmholt) [reply reference/7.1/Microsoft.PowerShell.Core/About/about_PowerShell_NuGet_Packages.md:6]: Ah that's where I copied the header from

**Comment by joeyaiello** [comment]
I agree there's a core problem that we don't have a well understood delineation between About topics and conceptual docs. 

We should chat offline @sdwheeler, and anyone else that wants to brainstorm ideas.

In the meantime, I saw this one got closed? Was there a related one that did get merged? I thought it was really useful content, we can handle the location later

**Pull request closed without merging**